### ISSUE TITLE
Retux has malware again! 

### ISSUE THREAD

**Issue by deonsebastiandk** [retux-game/retux#4]
Dear retux team, before I posted the issue 'retux has malware' issue and that time only a little portion of antiviruses scanned it as 'Unsafe' but today I downloaded the retux zip file for windows again and this time more antiviruses detect this as malware. Here is the virus total report - https://www.virustotal.com/gui/file/25b0424d77d457346a3545d0cb9d69109c0551d15bd1ebe40acc507ae7d129df/detection. Please remove these as fast as possible.

**Comment by onpon4** [comment]
That's exactly what it was at when I closed the previous issue. Please stop spamming this issue tracker and learn how antiviruses work. As I said on #3 already, this is clearly because of precautionary quarantines of behavior an antivirus considers "suspicious", and the antiviruses doing this are in the minority, with the only ones of any significance being McAfee and BitDefender.

Do not open another duplicate issue for this again unless you have real evidence of malware being in retux.exe. If you open another duplicate of #3 with this same standard of evidence, you will be blocked. I don't have the time or the spoons to constantly deal with false reports based on the false positives of a few antiviruses just because you don't understand how common of an occurrence that is. Please contact the developers of the antivirus that you use instead so they can investigate. From all the evidence I have available to me, they should find, as I did, that there is no actual threat present.

🕵️

**Comment by deonsebastiandk** [comment]
sorry , i won't do it again
